src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -56,6 +56,24 @@ if not [%NET46_PUBLISH_ERROR_LEVEL%]==[0] (
 	exit /b %NET46_PUBLISH_ERROR_LEVEL%
 )
 
+set ILASM_PROJECTJSON=%BUILDTOOLS_PACKAGE_DIR%\ilasm\project.json
```

Comment:
Given that the ILAsm package depends on a matching Runtime.CoreCLR package, how should I accomplish that, then? Adding it to the existing tool-runtime/project.json will cause an invalid version of CoreCLR to get pulled in and published with our tools.